tests/PeoplefinderRecordTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+require_once dirname(__FILE__) . '/../src/UNL/Peoplefinder/Record.php';
+
+class PeoplefinderRecordTest extends \PHPUnit_Framework_TestCase {
+    protected $record;
+
+    protected function setUp() {
+        $this->record = $this->getMockBuilder('UNL_Peoplefinder_Record')
+                                ->setMethods(array('getBuildings'))
+                                ->getMock();
+        $this->record->method('getBuildings')
+                        ->willReturn(array(
+            '420' => '420 University Terrace',
+            'ADMS' => 'Canfield Administration Building South',
+            'AVH' => 'Avery Hall',
+            '17PG' => '17th and R Parking Garage'
+        ));
+    }
+
+    public function testFormatterDoesNotThrowError() {
```

Comment:
What is going on here?  In what case would `formatPostalAddress()` throw an error?